index.js

```diff
@@ -24,6 +25,12 @@ const FRONTEND_URL =
   process.env.FRONTEND_DEPLOYED_URL || 'http://localhost:3000';
 const PORT = process.env.PORT || 5000;
 
+app.all('/*', function(req, res, next) {
```

Comment:
Looks Great! Though right here you seem to be adding duplicate code. app.all("/*)etc...

Reply:
Just removed the duplicate. I'll remove the app.all() as well